pkg/drivers/vfkit/vfkit.go

```diff
@@ -136,6 +136,49 @@ func (d *Driver) GetIP() (string, error) {
 	return d.IPAddress, nil
 }
 
+func writePidfile(pidfile string, pid int) error {
```

Comment:
there might be a util or lib that we are using in other parts of the code for writing or Locking PID files

Reply:
Using now the process package #20528, which uses the same method used in other packages for managing pids.